GNUmakefile

```diff
@@ -81,19 +77,19 @@ providerlint:
 
 tflint:
 	@echo "==> Checking Terraform code with tflint..."
-	@tflint --init
+	@go tool tflint --init
```

Comment:
This doesn't appear to actually run tflint, it just installs plugins - I'll open separate issue because running it gives a lot of errors right now